src/renderer/state.ts

```diff
@@ -82,6 +82,7 @@ export class AppState {
   @observable public isClearingConsoleOnRun = !!this.retrieve(
     'isClearingConsoleOnRun',
   );
+  @observable public isShowingTimestamp = !!this.retrieve('isShowingTimestamp');
```

Comment:
This needs to be followed by the default value, similar to the `?? true` in `isUsingSystemTheme` below.

Reply:
I'd also say we should have timestamps enabled by default - e.g. the option in settings is to _disable_ it. So the option might instead be something like `isDisablingTimestamps`

Reply:
Agree that the default value should be to show the timestamps.

*bikeshed*: I prefer the current name. All other things being equal, code should avoid negative logic because it leads to double-negative expressions e.g. expressing "are timestamps enabled?" as `!isDisablingTimestamps`.

Reply:
Thanks for the reviews! I'll take action on both of these suggestions 🙇